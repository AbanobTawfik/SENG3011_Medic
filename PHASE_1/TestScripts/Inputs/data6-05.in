#!/bin/sh
# invalid date 1

start_date=2015-01-01T00:00:00
end_date=2015-02-29T00:00:00
