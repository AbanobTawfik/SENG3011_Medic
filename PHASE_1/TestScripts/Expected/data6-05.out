{
    "meta": {
        "team_name": "Medics",
        "accessed_time": "REMOVED",
        "data_source": {
            "name": "CDC",
            "url": "https://www.cdc.gov/outbreaks/"
        }
    },
    "errors": {
        "end_date": "invalid value '2015-02-29T00:00:00'"
    }
}
